### ISSUE TITLE
Add subrecipe ingredients to main list

### ISSUE THREAD

**Issue by Gotanius** [TandoorRecipes/recipes#1993]
### Is your feature request related to a problem? Please describe.

When adding a (sub)recipe to a recipe the ingredients of that subrecipe aren't added tot ingredients list at the top of the main recipe. 

### Describe the solution you'd like

When adding a subrecipe add the ingredients to the top of the mainrecipe ingredients list with a header naming the subrecipe.

_pepper symbol_ Ingredients
ingredient 1 
ingredient 2 
ingredient 3 
etc..

_Name subrecipe_
Subrecipe ingredient 1
subrecipe ingredient 2 
etc...

**Comment by vabene1111** [comment]
Yes, not sure if there is an issue for this somewhere already... Will be done together with the sum setting

**Comment by chloec2** [comment]
I would like to take this issue!
